Resources/Textures/_Harmony/Structures/Machines/microwave.rsi/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by BoskiYourk for use in Harmony",
```

Comment:
CC-BY-SA-3.0 allows the content to be used, among other things, outside of Harmony. `for use in Harmony` might be misleading as restriction, which can't be done in this license